src/pages/Explorer/requests.js

```diff
@@ -0,0 +1,165 @@
+import { query } from 'webkit/api'
+
+export const getItems = ({
+  types = [],
+  range,
+  voted = false,
+  currentUserDataOnly = false,
+  page = 1,
+  pageSize = 20,
+} = {}) => {
+  const QUERYKEY = `getMost${voted ? 'Voted' : 'Recent'}`
+  const CURSOR = range ? `cursor: { type: AFTER, datetime: "utc_now-${range}" }` : ''
+
+  const projectWatchlist = types.includes('PROJECT_WATCHLIST')
+    ? `projectWatchlist{ 
+      id
+      title: name,
+      user {
+        avatarUrl
+        id
+        name
+        username
+      }
+      votes {
+        totalVotes
+      }
+      commentsCount
+      listItems {
+        project {
+          slug
+        }
+      }    
+    }`
```

Comment:
`[D-0-1]` Store the string in the variable that is outside of the query function.